src/idl_gen_rust.cpp

```diff
@@ -131,8 +131,6 @@ static std::set<std::string> RustKeywords() {
     // Terms that we use ourselves
     "follow",
     "push",
-    "size",
-    "alignment",
```

Comment:
no need to change anything else in idl_gen_rust? Is that because they're static methods of the `Push` trait and `Self::size()` doesn't conflict with `foo.size()`?

Reply:
I thought that would be the case, because I thought Self::size would default to the Push implementation, but I guess that's not the case. I updated the generated code to use <Self as flatbuffers::Push>::size() instead.